spec/importer/importer_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'rails_helper'
+RSpec.describe Importer do
+  subject(:importer) { described_class.new(parser: parser) }
+  let(:parser) { Darlingtonia::CsvParser.new(file: file) }
+  let(:file) { File.open('spec/fixtures/example.csv') }
+
+  it "can create an Importer" do
+    expect(importer).to be_instance_of described_class
+  end
+
+  it "inherits from the Darlingtonia Importer" do
+    expect(described_class).to be < Darlingtonia::Importer
+  end
+
+  it "creates three works from a three-row csv" do
+    expect { importer.import }.to change { Etd.count }.by 3
+    3.times do
```

Comment:
Better to just add the `:clean` tag?

Reply:
@jenlindner and I talked about this in slack, decided this approach is :ok_hand:, and I'm :+1: